api/data/seattle_db.go

```diff
@@ -335,7 +350,7 @@ func (c *Client) SeattleFuzzySearchByLastName(lastName string) ([]*SeattleOffice
 func seattleMarshalOfficerRows(rows pgx.Rows) ([]*SeattleOfficer, error) {
 	officers := []*SeattleOfficer{}
 	for rows.Next() {
-		ofc := SeattleOfficer{}
+		ofc := seattleOfficer{}
 		err := rows.Scan(
 			&ofc.Date,
 			&ofc.Badge,
```

Comment:
How does this stuff work here?

Reply:
`ofc := seattleOfficer{}` is variable instantiation, while `rows.Scan` hydrates the `ofc` struct with data from the db

Reply:
Ooohhhh I see, that makes sense, thanks!